pom.xml

```diff
@@ -257,7 +257,7 @@
     <version.org.hamcrest>1.3</version.org.hamcrest>
     <version.junit>4.12</version.junit>
     <!-- keep in sync with version.com.datastax.cassandra -->
```

Comment:
They were not equal before (if so, one property would have been enough), so I think it is ok like  this